config/locales/en/casework.yml

```diff
@@ -39,13 +40,11 @@ en:
       no_change_since_last_update:
         - The application on MAAT has not changed.
         - Make any required amendments on MAAT using the MAAT ID %{maat_id}. 
-      already_sent_back: This application was already sent back to the provider
-      already_completed: This application was already marked as complete
+      already_reviewed: This application was already completed
```

Comment:
Not sure I understand this change, we no longer need this message? `This application was already sent back to the provider` ?

Reply:
These messages are unlikely to occur because the UI prevents them. I want to simplify the code to use a "you can't review something that's already been reviewed error", rather the more specific errors.

Reply:
Agreed to change 'completed' to 'reviewed'